pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -471,6 +471,7 @@ self: super: {
   });
 
   youcompleteme = super.youcompleteme.overrideAttrs(old: {
+    disabled = !python.isPy3k;
```

Comment:
pretty sure this is unnecessary since you made the change in the python-module as well

Reply:
I disabled python2 support in both `ycmd` and `youcompleteme`, as they're two separate modules, that link back to two separate repositories with python2 support explicitly dropped in each.

I'm fine with whatever, though, and if it makes more sense to remove the `disabled` bit here, I'd be happy to do so.

Reply:
I'm not sure if this does anything, the `disabled = ...` is specific to `buildPython{Application,Package}`

Reply:
Sounds good.  Didn't realize it was `buildPython` specific.